### PR TITLE
Satellite remote sensing tuto

### PR DESCRIPTION
**Pull request by Marie59**
Initiating the PR for the tutorial for conducting a biodiversity study on Sentinel 2 data.

### CODE REVIEW

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+
+> <hands-on-title>Hands-on: Principal components analysis for remote sensing data</hands-on-title>
+>
+> 1. {% tool [Compute a PCA](toolshed.g2.bx.psu.edu/repos/ecology/srs_pca/srs_pca/0.0.1) %} with the following parameters:
+>    - {% icon param-select %} *"In which format are your data ?"*: 'The data you are using are in a zip folder Reflectance'
+>      - {% icon param-file %} *"Input data"*: `Reflectance` (output of **Preprocessing sentinel 2 data** {% icon tool %})
+>    - {% icon param-select %} *"Do you want to do a PCA or a SPCA ?"*: 'PCA'
+>
+>
+>    > <tip-title>Tip: If you want to use your own files ENVI BIL<h/tip-title>
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:261]:
```suggestion
>    > <tip-title>Tip: If you want to use your own files ENVI BIL</tip-title>
```

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+>    >
+>    {: .comment}
+>
+{: .hands_on}
+
+
+![ndvi](../../images/remote_sensing/ndvi.png "Normalized different vegetation index tabular")
+
+
+> <detail-title>Spectral indices and Essential Biodiversity Variables</detail-title>
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:401]:
```suggestion
> <details-title>Spectral indices and Essential Biodiversity Variables</details-title>
```

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+>
+{: .hands_on}
+
+# Preparing data
+
+Using preprocS2 R package, this step provides a unique tool to read, crop, resample the original image directory, and write it as a raster stack
+
+## **Preprocessing sentinel 2 data**
+
+> <hands-on-title>Hands-on: Preprocess</hands-on-title>
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:120]:
```suggestion
> <hands-on-title>Preprocess</hands-on-title>
```

The `Hands-on:`, `Question:` etc part are already added automatically, so you can leave them out here and only add what you want to appear after that (or leave empty if nothing)

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+>    {: .comment}
+>
+{: .hands_on}
+
+![Preprocessing outputs](../../images/remote_sensing/preproc.png "Preprocessing outputs")
+
+
+> <question-title>Question</question-title>
+>
+> 1. What are the files you are interested in for the folowing tools ?
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:140]:
```suggestion
> 1. What are the files you are interested in for the following tools ?
```

**Review by shiltemann — COMMENTED** [review]
Thanks @Marie59! Very cool!

just some minor formatting fixes below :) (you can use the "commit suggestion" button to accept suggested changes)

**Comment by shiltemann** [comment]
Hi @Marie59! I've added some final tweaks. It looks good to me! I see there are still some TODOs written in there, so just let me know when you are completely ready and I will do final review and merge it :)

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+>    >
+>    >
+>    > ***TODO***: *Here again check that the "Input raster" datatype is bil and that "Input raster header" datatype is hdr*
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+![Global biodiveristy tabular](../../images/remote_sensing/rasterdivtab.png "Global biodiveristy tabular")
+![Global biodiveristy graph](../../images/remote_sensing/rasterdiv.png "Global biodiveristy graph")
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:229]: oh, and could you update the alt texts (the bit between the `[..]` ) for the images to be a bit more descriptive? Imagine you are describing what is in the image to a blind person

Comment (hexylena) [reply topics/ecology/tutorials/remote-sensing/tutorial.md:229]: We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

**Comment by yvanlebras** [comment]
@Marie59 it seems there is some TODO on remote sensing tuto notably. Can you look at it and update before asking @shiltemann to review and merge ? Moreover, It would be nice to propose an example datafile as input for the tuto. Don't remember if we were finding a good example data not too big?

**Comment by Marie59** [comment]
> @Marie59 it seems there is some TODO on remote sensing tuto notably. Can you look at it and update before asking @shiltemann to review and merge ? Moreover, It would be nice to propose an example datafile as input for the tuto. Don't remember if we were finding a good example data not too big?

I have one or 2 datafiles not that big but still it would take 2 days I think to do the full analysis

**Comment by Marie59** [comment]
Should I open a new PR and close this one ? I don't know how to resolve the conflicts here

**Comment by hexylena** [comment]
I can try to help with that

**Comment by hexylena** [comment]
@Marie59 why does the last commit delete the `champs-blocs` directory? I believe this is the main cause of the conflict. Is that tutorial no longer needed or something?

**Comment by hexylena** [comment]
Ahh no worries, that makes it easier to resolve. :)

**Review by hexylena — DISMISSED** [review]

**Comment by hexylena** [comment]
Just the same workflow issue (the test input missing/creator details.)

> I have one or 2 datafiles not that big but still it would take 2 days I think to do the full analysis

I think even that is good, better to have an example dataset than none.

**Comment by Marie59** [comment]
Ooooh could it be finally ready ?! 
Thank you for all your help @hexylena @shiltemann and @yvanlebras  !!

**Comment by shiltemann** [comment]
Thanks, it looks good! 

Just a question about the time, you said it takes 2 days to complete the tutorial or did I misunderstand? Can we maybe add a warning box to say the analysis will take 2 days so people don't wait around for it? And maybe update it in the time estimate metadata? (it currently says 2 hours). 

Which step is it that takes this long? is it possible/would it make sense to provide the outputs from that 2-day step as well (e.g. on Zenodo), so that learners or instructors can continue with the next steps without waiting 2 days?

**Comment by Marie59** [comment]
Yes, of course I will add a warning and change the time. It's one of the last step that takes a lot of time. I can add one of the output to show but the output is not needed for the following steps. So, the users can easily just skip it or continue with the rest.

**Comment by shiltemann** [comment]
I think it would still be nice to show the output so people can view it without waiting the 2 days, if it's  not too much trouble to do that.

**Comment by Marie59** [comment]
I put to photos for it. It's the beta part is it okay or do we need more ?

**Comment by Marie59** [comment]
I have one last photo too add (I am waiting for the tool to run to put it)

**Comment by shiltemann** [comment]
> I put two photos for it. It's the map biodiversity part is it okay or do we need more ?

Just whatever you think is needed so that they can evaluate the results of the tool without having to wait the 2 days :) :+1:

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+questions:
+- How to get spectral and biodiversity indicators from remote sensing data ?
+- Which kind of ecosystem are you studying ? global or canopy data ?
+objectives:
+- Getting Sentinel 2 data and reformatting them in a generalized way
+- Computing spectral indices such as the NDVI
+- Calculating and visualizing biodiversity indicators
+- Comparing with in-situ data
+time_estimation: 48H
+key_points:
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:15]: Last request: Key points should be conclusions from the tutorial that are important to remember. Think about what the lessons are you want learners to have learned from this tutorial. So can you rephrase them to be full sentences? Things like "It is always important to do X because ..", "Always keep in mind X when interpreting Y", "X is a method to do Y", etc.

**Comment by shiltemann** [comment]
perfect, thanks!

File: topics/ecology/tutorials/remote-sensing/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+- Remote sensing data can be usefull to find hotspot of biodiversity.
```
Comment (shiltemann) [inline topics/ecology/tutorials/remote-sensing/tutorial.md:16]:
```suggestion
- Remote sensing data can be useful to find hotspots of biodiversity.
```

**Review by shiltemann — APPROVED** [review]

**Comment by Marie59** [comment]
Youhou!! Thank you for your help !!
